pkg/kubelet/pleg/generic.go

```diff
@@ -81,6 +82,10 @@ const (
 	// relisting time, which can vary significantly. Set a conservative
 	// threshold to avoid flipping between healthy and unhealthy.
 	relistThreshold = 3 * time.Minute
+
+	// Too many pods in node scene, relist take long time to update pod cache
+	// which result in node not ready. parallelUpdatePodNum means parallel num to update pod cache.
+	parallelUpdateCacheNum = 50
```

Comment:
how did you come with that number? should it be user settable? what's the implications on less powerful nodes?

Reply:
For this number, I think we can talk about. :)